src/index.scss

```diff
@@ -0,0 +1 @@
+@import 'application';
```

Comment:
same comment, this should be on `scss` folder named as `application.scss` this file should contain all general styles overrides like `body` and the global sass imports like `wolox-equalizer`